### PR TITLE
Ratings model

### PR DESCRIPTION
**Pull request by xuorig**
@cjoudrey

### CODE REVIEW

File: app/models/film.rb [hunk 1]
```diff
@@ -8,4 +8,7 @@ class Film < ApplicationRecord
     join_table: 'films_people',
     foreign_key: 'film_id',
     association_foreign_key: 'person_id'
+
+  has_many :ratings
+  has_many :users, through: :ratings
```
Comment (cjoudrey) [inline app/models/film.rb]: `:raters`? if that's a word lol.

File: app/models/rating.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Rating < ApplicationRecord
+  belongs_to :user
+  belongs_to :film
+
+  validates_inclusion_of :number, :in => 0..5
```
Comment (cjoudrey) [inline app/models/rating.rb]: Should be able to do: `in: 0..5`

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  has_many :ratings
+  has_many :films, through: :ratings
```
Comment (cjoudrey) [inline app/models/user.rb]: `:rated_films`?

**Review by cjoudrey — APPROVED** [review]

File: app/models/film.rb (hunk 1 above)
Comment (xuorig) [reply app/models/film.rb]: i guess `critics` :P

Comment (cjoudrey) [reply app/models/film.rb]: Ah! Yes!

It'll allow us to easily add `field :critics` in GraphQL type if we name the association correctly.

**Pull request merged**